### PR TITLE
[Edit] CSS [Edit] CSS Borders: border-width

### LINKED ISSUE #3645: [Edit] CSS [Edit] CSS Borders: border-width
### Type of Edit (select all that apply)

Add new content (definitions, codeblocks, etc.)

### Description (optional)

[PLEASE DO NOT WORK ON THIS ISSUE UNTIL YOU HAVE BEEN ASSIGNED] We would like an edit to a term within CSS Borders. The term entry lives in docs/content/css/concepts/borders/terms/border-width/border-width.md

Please refer to the [term entry templates](https://github.com/Codecademy/docs/blob/main/documentation/term-entry-template.md), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

Please:
1) Add supplemental media (preferred .png or .gif, <1MB).
2) Media (images, gifs) should accurately represent the existing code blocks in the example sections and fit in-line. ex: [See last image here](https://www.codecademy.com/resources/docs/html/attributes/title)

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [x] Labels added
- [x] Issue is assigned

### PR DESCRIPTION
**Pull request by YashSalunke12**
### Description
closes #3645 

- Added more values in syntax.
- Illustrated values of syntax in proper bullet points for better understanding.
- Added images for examples.

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [X ] All writings are my own.
- [ X] My entry follows the Codecademy Docs style guide.
- [X ] My changes generate no new warnings.
- [X ] I have performed a self-review of my own writing and code.
- [ X] I have checked my entry and corrected any misspellings.
- [ X] I have made corresponding changes to the documentation if needed.
- [X ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by YashSalunke12** [comment]
@SSwiniarski could you take a look, please?

File: content/css/concepts/borders/terms/border-width/border-width.md [hunk 1]
```diff
@@ -18,15 +18,46 @@ Shorthand property that sets the width for the overall border of an element or f
...
+- When four values are specified, the widths apply to the top, right, bottom, and left in that order (clockwise).
 
-**Note:** Providing one value will effect the width of all four borders. Providing two values will result in the first value setting the width of top and bottom borders and the second value setting the width of left and right. Three values will result in the first value applied to the top, the second value to the left and right, and the third value applied to the bottom. Four values will apply to top, right, bottom and left.
+### values
+`<line-width>`
+
+Defines the width of the border, either as an explicit nonnegative `<length>` or a keyword. If it's a keyword, it must be one of the following values:
+- `thin`
+- `medium`
+- `thick`
```
Comment (SSwiniarski) [inline content/css/concepts/borders/terms/border-width/border-width.md:60]:
```suggestion
```
I don't think this section is necessary.

File: content/css/concepts/borders/terms/border-width/border-width.md [hunk 2]
```diff
@@ -51,6 +85,9 @@ h1 {
   border-width: 4px 2px;
 }
 ```
+The following image demonstrates the code block above:
+
+![CSS border-top Example 1](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-borders-border-width-example1.png)
```
Comment (SSwiniarski) [inline content/css/concepts/borders/terms/border-width/border-width.md:90]:
```suggestion
![CSS border-top Example 2](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-borders-border-width-example2.png)
```

File: content/css/concepts/borders/terms/border-width/border-width.md [hunk 3]
```diff
@@ -63,6 +100,9 @@ h1 {
   border-width: 10px medium thick;
 }
 ```
+The following image demonstrates the code block above:
+
+![CSS border-top Example 1](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-borders-border-width-example1.png)
```
Comment (SSwiniarski) [inline content/css/concepts/borders/terms/border-width/border-width.md:105]:
```suggestion
![CSS border-top Example 3](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-borders-border-width-example3.png)
```

File: content/css/concepts/borders/terms/border-width/border-width.md [hunk 4]
```diff
@@ -75,3 +115,6 @@ h1 {
   border-width: 10px thin 20px thick;
 }
 ```
+The following image demonstrates the code block above:
+
+![CSS border-top Example 1](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-borders-border-width-example1.png)
```
Comment (SSwiniarski) [inline content/css/concepts/borders/terms/border-width/border-width.md:120]:
```suggestion
![CSS border-top Example 4](https://raw.githubusercontent.com/Codecademy/docs/main/media/css-borders-border-width-example4.png)
```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@YashSalunke12 I did a first review and left a few comments.

File: content/css/concepts/borders/terms/border-width/border-width.md (hunk 1 above)
Comment (YashSalunke12) [reply content/css/concepts/borders/terms/border-width/border-width.md]: okay I've removed this section.

**Comment by YashSalunke12** [comment]
@SSwiniarski  ls it okay now?

File: content/css/concepts/borders/terms/border-width/border-width.md (hunk 2 above)
Comment (YashSalunke12) [reply content/css/concepts/borders/terms/border-width/border-width.md:83]: Oh sorry, I forgot to change this.

**Review by SSwiniarski — APPROVED** [review]
@YashSalunke12 Looks good for a second review. Note you still need to sign the [CLA](https://cla-assistant.io/Codecademy/docs?pullRequest=3689).

**Review by caupolicandiaz — APPROVED** [review]
@YashSalunke12 @SSwiniarski LGTM

**Comment by caupolicandiaz** [comment]
@YashSalunke12 Looks the CLA has not been signed

**Comment by YashSalunke12** [comment]
@SSwiniarski @caupolicandiaz done with signing CLA

**Pull request merged**